pkg/docker-image/README.md

```diff
@@ -0,0 +1,39 @@
+# Official Urbit Docker Image
+
+This is the official Docker image for [Urbit](https://urbit.org). 
+
+Urbit is a clean-slate OS and network for the 21st century.
+
+## Using
+
+To use this image, you should mount a volume with a keyfile, comet file, or existing pier at `/urbit`, and map ports
+as described below.
+
+### Volume Mount
+This image expects a volume mounted at `/urbit`. This volume should initially obtain one of
+
+- A keyfile `<shipname>.key` for a galaxy, star, planet, or moon. See the setup instructions for Urbit for information on [obtaining a keyfile](https://urbit.org/using/install/). 
+  * e.g. `sampel-palnet.key` for the planet `sampel-palnet`.
+  to urbit to boot a ship from an existing pier, and in most cases (other than comets) will be named for the ship.
```

Comment:
I'm having trouble understanding what exactly is meant by this sentence—could this be rephrased?

Reply:
I think this was a spurious paste as a result of writing docs too late at night. I've removed it.